js/utils.js

```diff
@@ -974,6 +974,56 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
     return Promise.resolve(records);
   }
 
+  /**
+   * Sort items by field witch user have selected
+   * 
+   * @param {Object} options - object with settings for sort list items
+   *        keys: 
+   *          sortField {String} - name of the field to sort
+   *          sortOrder {String} - sort order of
+   *          records {Array} - array of records to sort
+   * @return {Array} - sorted by field array
+   */
+  function sortByField(options) {
+    // If user doesn't set sorting do nothing
+    if (!options.sortField) {
+      return options.records;
+    }
+
+    var records = options.records;
+    var orderBy = options.sortOrder;
```

Comment:
`orderBy` means something else entirely when we quite commonly use `_.orderBy()`

```suggestion
    var sortOrder = options.sortOrder;
```